src/uPlot.js

```diff
@@ -132,6 +132,8 @@ export default function uPlot(opts, data) {
 		if (!(key in scales)) {
 			scales[key] = {
 				auto: true,
+				rawMin:  inf,
+				rawMax: -inf,
```

Comment:
i'd prefer to re-compute these on demand from the series' min/max caches. it's cheap and reduces cache maintenance in more places than necessary.

Reply:
Nice, I'll do that